data/json/items/gun/223.json

```diff
@@ -45,6 +45,29 @@
     "min_cycle_recoil": 1350,
     "magazines": [ [ "223", [ "stanag30", "stanag50", "survivor223mag" ] ] ]
   },
+  {
+    "id": "famas",
+    "copy-from": "rifle_auto",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "FAMAS G2" },
```

Comment:
```suggestion
    "name": { "str": "MAS .223", "str_pl": "MAS .223" },
```

Reply:
Why specify the caliber? The already will automatically do that.

Reply:
That's its name.

Reply:
That option is on by default, so most people will see it. I don't particularly have an opinion on this, just a heads up.

Reply:
Maybe `MAS 223` would be better then. hmm.